checkstyle.json

```diff
@@ -35,7 +35,8 @@
 					"STATIC",
 					"MACRO",
 					"INLINE",
-					"DYNAMIC"
+					"DYNAMIC",
+					"FINAL"
```

Comment:
holy crap this has been driving me nuts, I was told this wasn't possible

Reply:
Yeah its not even shown in the docs, I had to go into the default config json on the repository to find it

Reply:
it seems like documentation for ModifierOrder check wasn't properly updated when Haxe 4 / checkstyle 2.5.0 came out.
`FINAL` had been in checkstyle for a bit longer, but was hidden behind a `#if haxe4` conditional. I should've updated documentation when I moved to Haxe 4 releases, but it seems I've missed it. sorry.

luckily the default config is generated automatically so it always contains whatever features and settings are available and active in any given version.

I've updated docs to include `FINAL` and the new message format.

Reply:
Sweet, thank you!